nmosnode/registry.py

```diff
@@ -460,16 +460,24 @@ def list_resource(self, type, api_version="v1.0"):
                     ))
         return response
 
+    def _len_resource(self, type):
+        if not type in self.permitted_resources:
+            return RES_UNSUPPORTED
+        response = 0
+        for name in self.services:
+            response += len(self.services[name]["resource"][type])
+        return response
+
     def _update_mdns(self, type):
-        items = self.list_resource(type)
-        if not isinstance(items, dict):
-            return
-        if len(items) == 1:
+        if not type in self.permitted_resources:
+            return RES_UNSUPPORTED
+        items = self._len_resource(type)
```

Comment:
Being super picky here, but if you do end up making some tweaks could this be "numItems" or something to that effect?